### DISCUSSION TITLE
PyPI package

### DISCUSSION THREAD

**Discussion by steinnymir** [OpenCOMPES/sed#73] [Ideas]
At some point we would like to package this project and publish it on PiPI.
I had a look and `sed` does not look like a unique, nor a memorable or self-explanatory name.

Do we want to think of a different name to use for the PiPI package? 
One option could be `sed-binnning', or `clean-tables` as they used to be called in the very beginning... 
It also comes down to what the main scope of this package would be, and if we are going to make other projects spinning out of here for the possible different applications (i.e. sed-pes).

Any ideas or suggestions?
@zainsohail04 @rettigl @daviddoji

**Comment by rettigl** [comment]
I had similar problems finding the repo, so changing the name is probably not a bad idea.

Regarding the general outline, we should discuss on Monday where we want to go with this. Really helpful would be an interface that wraps the whole workflow into a class, and includes and interfaces the individual pieces, like dataframe generation, calulation of conversion parameters, and finally binning, all with documentation and meta data generation. This would, as I see it, however be another still to develop package.

**Reply by zain-sohail** [reply]
Dataframe generation [preprocessing part] was decided to be a seperate pacakage, and it can naturally be integrated into sed through a simple interface as you suggested. I agree that we should discuss the architecture [maybe a flow chart] and have it readily available to view perhaps on github discussions.

**Comment by zain-sohail** [comment]
Perhaps sed-processor is more fitting? Since sed is supposed to not just bin but do other processing tasks.
